includes/admin/stripe-settings.php

```diff
@@ -291,44 +291,38 @@
 
-	// no longer needed in the new settings.
-	unset( $stripe_settings['express_checkout_button_branded_type'] );
-	unset( $stripe_settings['express_checkout_button_height'] );
-	unset( $stripe_settings['express_checkout_button_label'] );
-	// injecting some of the new options.
-	$stripe_settings['express_checkout_button_type']['options']['default'] = __( 'Only icon', 'woocommerce-gateway-stripe' );
-	$stripe_settings['express_checkout_button_type']['options']['book']    = __( 'Book', 'woocommerce-gateway-stripe' );
-	// no longer valid options.
-	unset( $stripe_settings['express_checkout_button_type']['options']['branded'] );
-	unset( $stripe_settings['express_checkout_button_type']['options']['custom'] );
-} else {
-	unset( $stripe_settings['express_checkout_button_size'] );
-}
+// no longer needed in the new settings.
+unset( $stripe_settings['express_checkout_button_branded_type'] );
+unset( $stripe_settings['express_checkout_button_height'] );
+unset( $stripe_settings['express_checkout_button_label'] );
+// injecting some of the new options.
+$stripe_settings['express_checkout_button_type']['options']['default'] = __( 'Only icon', 'woocommerce-gateway-stripe' );
+$stripe_settings['express_checkout_button_type']['options']['book']    = __( 'Book', 'woocommerce-gateway-stripe' );
+// no longer valid options.
+unset( $stripe_settings['express_checkout_button_type']['options']['branded'] );
+unset( $stripe_settings['express_checkout_button_type']['options']['custom'] );
 
-if ( WC_Stripe_Feature_Flags::is_upe_preview_enabled() ) {
-	$upe_settings = [
-		WC_Stripe_Feature_Flags::UPE_CHECKOUT_FEATURE_ATTRIBUTE_NAME => [
-			'title'       => __( 'New checkout experience', 'woocommerce-gateway-stripe' ),
-			'label'       => sprintf(
-				/* translators: 1) br tag 2) Stripe anchor tag 3) Apple anchor tag 4) Stripe dashboard opening anchor tag 5) Stripe dashboard closing anchor tag */
-				__( 'Try the new payment experience (Early access) %1$sGet early access to a new, smarter payment experience on checkout and let us know what you think by %2$s. We recommend this feature for experienced merchants as the functionality is currently limited. %3$s', 'woocommerce-gateway-stripe' ),
-				'<br />',
-				'<a href="https://woocommerce.survey.fm/woocommerce-stripe-upe-opt-out-survey" target="_blank">submitting your feedback</a>',
-				'<a href="https://woocommerce.com/document/stripe/admin-experience/new-checkout-experience/" target="_blank">Learn more</a>'
-			),
-			'type'        => 'checkbox',
-			'description' => __( 'New checkout experience allows you to manage all payment methods on one screen and display them to customers based on their currency and location.', 'woocommerce-gateway-stripe' ),
-			'default'     => 'no',
-			'desc_tip'    => true,
-		],
-	];
-	// Insert UPE options below the 'logging' setting.
-	$stripe_settings = array_merge( $stripe_settings, $upe_settings );
-}
+$upe_settings = [
```

Comment:
We should remove this; it's not being used, based on the title (`New checkout experience`) and the description, this seems to be from the old settings page, before we made the "New Checkout" the standard checkout.

Reply:
Good catch! Removed it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4760/commits/16b2967ef8a11fd6ef7c9ddae764cec18d65b94c